modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -764,6 +768,28 @@ public List<ObjectDefinition> getSystemObjectDefinitions() {
 		return objectDefinitionPersistence.findBySystem(true);
 	}
 
+	@Transactional(propagation = Propagation.REQUIRES_NEW)
+	public long getUniqueComposedKeyObjectValidationRuleDslQueryCount(
```

Comment:
This method should be inside `ObjectEntryLocalServiceImpl`, no? There we perform queries over object entries.

Move this method to there and rename it to `getObjectEntryCount`.

Reply:
Great catch!
Thank you!